adapters/sspBC/sspbc.go

```diff
@@ -128,6 +128,7 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 		Method: http.MethodPost,
 		Uri:    requestURL.String(),
 		Body:   requestJSON,
+		ImpIDs: getImpIDs(formattedRequest.Imp),
```

Comment:
Forgot to use openrtb_ext.GetImpIDs here. Please delete the package specific method.

Reply:
Hi @SyntaxNode ,
it is quite confusing as imp.Id is set into imp.tagid and again it is recovered in makeBids. imp.Id should be used or imp.TagId?

Reply:
Also imp.ID depends on `extSSp.id` and `requestype`

Reply:
Good call. I missed that. I think we're good here with that context.